pyproject.toml

```diff
@@ -9,18 +9,18 @@ packages = [{include = "iwf"}]
 [tool.poetry.dependencies]
 python = "^3.9"
 python-dateutil = { version = "^2.8.2", python = "<3.11" }
+httpx = "0.28.1"
```

Comment:
When users install the iwf-python-sdk package, and they try to use it, they will get errors around httpx functions missing.

That is because we were only declaring httpx as a dev dependency, and not as a package dependency.

With these changes, when users require iwf-python-sdk into their project, they won't have to require any additional packages.

Reply:
And the `httpx` dependency has been updated to the latest version